frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1543,47 +1622,128 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
+                          <Td
+                            key={`${row.props.key}-select`}
+                            select={{
+                              rowIndex,
+                              onSelect: onSelectCallback(),
+                              isSelected: selected[row.props.key],
+                              isDisabled: row.disableSelection,
+                            }}
+                          />
+                        )}
+                        {row?.cells?.map((cell, cellIndex) => {
+                          // setup row props, including celltransforms
+                          const transforms: ITransform[] | undefined = selectedSortedCols[cellIndex]?.cellTransforms
+                          const iTransformCellProps = transforms
+                            ? mergeProps(
+                                ...transforms.map((transform: ITransform) =>
+                                  transform(row, {
+                                    rowIndex,
+                                  })
+                                )
+                              )
+                            : []
+                          const isMenuKebab = isKebabMenu(cell)
+                          const rowProps = {
+                            dataLabel: isMenuKebab ? undefined : columns[cellIndex].header,
+                            ...iTransformCellProps,
+                            className: isMenuKebab ? rowActionClass + ' ' + iTransformCellProps?.className : '',
+                          }
+                          return (
+                            <Td key={`row-${rowIndex}-cell-${cellIndex}`} {...rowProps}>
+                              {renderCellContent(cell)}
+                            </Td>
+                          )
+                        })}
+                        {(!!actionResolver || actions.length > 0) && (
+                          <Td isActionCell>
+                            {((!!actionResolver && actionResolver?.(row, { rowIndex }).length > 0) ||
+                              actions.length > 0) && (
+                              <ActionsColumn
+                                items={actionResolver ? actionResolver(row, { rowIndex }) : actions}
+                                actionsToggle={actionsToggle}
+                                extraData={{ rowIndex }}
+                                rowData={row}
+                              />
+                            )}
+                          </Td>
+                        )}
+                      </Tr>
+                      {addedSubRowCount > 0 && (
+                        <Tr isExpanded={row.isOpen} key={addedSubRows[rowIndex]?.props?.key}>
+                          {/* include spacing for expandable and selection columns in subrow */}
+                          {onCollapse && <Td />}
+                          {hasSelectionColumn && <Td />}
+                          <Td key={addedSubRows[rowIndex]?.props?.key} colSpan={9}>
```

Comment:
Please do not hard-code `colSpan` here.

Reply:
You need to calculate the correct `colSpan` here based on the number of columns currently displayed. Test with the Policies table, where you can vary the number of columns that are currently used.

<img width="2552" alt="image" src="https://github.com/user-attachments/assets/122af38e-e4df-4c99-9135-1024caa631b7" />

Reply:
Using the `addedSubRowCount` for the number of sub-rows (sub-columns) in each row, which aligns with this PF guidance as well: https://www.patternfly.org/components/table/#nested-column-headers

Thanks!

Reply:
\*\*BUZZER NOISE\*\* Try again 😕

https://github.com/user-attachments/assets/979da3a3-fecc-4d97-b0e4-07d17fb90274

Reply:
Dumb mistake!
Though I'm noticing now that the # of subRows used as a colSpan doesn't quite get us to the behavior of the current table. I'm thinking the # of items in column array might be a better approximation. We also have no great way calculating the number of subRows right now (except maybe by taking advantage of the exportSubrow array, which also seems hacky). We could pass them in, but that also feels a bit too involved.